kuzzleio/kuzzle-device-manager
# Replace nodemon with ergol

## What does this PR do ?
Replace nodemon with the homemade hot reloader [ergol](https://github.com/kuzzleio/ergol)

### How should this be manually tested?

Clone and run kuzzle-iot-platform, pick a random file in the lib directory, add anything and watch your terminal: the project should restart.
